ci-operator/step-registry/aro-e2e/classic/aro-e2e-classic-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+workflow:
+  as: aro-e2e-classic
+  steps:
+    pre:
+      - chain: aro-e2e-config
+      - ref: aro-e2e-classic-bootstrap-create
+      - ref: aro-e2e-classic-cluster-create
+    test:
+      - ref: aro-e2e-classic-test
+    post:
+      - ref: aro-e2e-classic-cluster-delete
+      - ref: aro-e2e-classic-bootstrap-delete
+    env:
+      AZURE_SUBSCRIPTION_ID: "fe16a035-e540-4ab7-80d9-373fa9a3d6ae"
+      AZURE_LOCATION: "eastus"
+      RP_ENDPOINT: "https://4.255.86.56:443"
+      ARO_VERSION: "4.14.33"
+      ARO_VERSION_OPENSHIFT_PULLSPEC: "quay.io/openshift-release-dev/ocp-release@sha256:40ba8c540d16a97a6b629232a5a16da64fca655c4b83b734aa28415b5c708acf"
+      ARO_VERSION_INSTALLER_PULLSPEC: "ocpciarotest.azurecr.io/aro-installer:release-4.14"
```

Comment:
Is it not possible to use an arbitrary release payload yet?

Reply:
I want to keep it simple in the first iteration, but in a follow up the  plan is to get this from the `releases` config in the actual test.